server/gitrest/packages/gitrest-base/src/utils/isomorphicgitManager.ts

```diff
@@ -291,7 +291,7 @@ export class IsomorphicGitRepositoryManager implements IRepositoryManager {
                     [BaseGitRestTelemetryProperties.ref]: refId,
                 },
                 err);
-            throw new NetworkError(500, "Unable to get ref.");
+            throw new NetworkError(400, "Unable to get ref.");
```

Comment:
Can we add a comment here explaining that GitManager.getRef relies on a `404 || 400` error code to return null, which is expected in some code paths like Scribe?